packages/client/src/services/MetricsService.ts

```diff
@@ -1,18 +1,20 @@
 import axios from 'axios';
+import log from '../util/LoggerFactory';
 
 export default class MetricsService {
-  async sendMetric(metricName: string, path: string): Promise<void> {
-    try {
-      await axios.post(`${process.env.PUBLIC_URL}/metrics`, {
-        name: `${metricName}`,
-        dimensions: {
-          url: `${path}`
-        }
-      })
+  async sendMetric(metricName: string, path?: string): Promise<void> {
+    let data: KvMap<object | string> = {
+      name: `${metricName}`
+    };
+    if (path) {
+      data['dimensions'] = {
+        url: `${path}`
+      };
     }
-    catch (e) {
-      console.log(`unable to send metric: ${metricName} and dimension: ${path}`);
-      console.log(e)
+    try {
+      await axios.post(`${process.env.PUBLIC_URL}/metrics`, data);
```

Comment:
You don't need to await this since your not doing anything with the results, so this whole method can just be a normal non-async method.